lib/node_modules/@stdlib/array/base/mskfilter-map/README.md

```diff
@@ -0,0 +1,173 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# mskfilterMap
+
+> Apply a mask to a provided input array and return a new array after applying a mapping function.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var mskfilterMap = require( '@stdlib/array/base/mskfilter-map' );
+```
+
+#### mskfilterMap( x, mask, clbk\[, thisArg] )
+
+Applies a mask to a provided input array and returns a new array after applying a mapping function.
+
+```javascript
+function clbk( value ) {
+    return value * 2;
+}
+
+var x = [ 1, 2, 3, 4 ];
+
+var y = mskfilterMap( x, [ 0, 1, 0, 1 ], clbk );
```

Comment:
```suggestion
var x = [ 1, 2, 3, 4 ];
var m = [ 0, 1, 0, 1 ];

var y = mskfilterMap( x, m, clbk );
```

The idea behind these changes is to make it visually more obvious what values are masked and to minimize eye movement when interpreting output results.